exercises/rotational-cipher/src/lib.rs

```diff
@@ -1 +1,7 @@
-
+pub fn rotate(input: &str, key: u8) -> String {
```

Comment:
None of the tests shift by negative keys, but there are plenty of people who would try shifting by `-1` before thinking of using `25`. I'd make `key` an `i8`, just as a hint to the students.